setup.py

```diff
@@ -13,7 +13,7 @@ def load_readme():
 def load_about():
```

Comment:
Since this plugin uses the `drydock-multipurpose-jobs` patch, should we indicate that this is only usable if drydock is installed?